addons/survey/models/survey_question.py

```diff
@@ -275,6 +280,49 @@ def _compute_validation_required(self):
             if not question.validation_required or question.question_type not in ['char_box', 'numerical_box', 'date', 'datetime']:
                 question.validation_required = False
 
+    @api.depends('is_conditional', 'survey_id', 'survey_id.question_ids', 'triggering_question_id')
+    def _compute_allowed_triggering_question_ids(self):
+        """ This method is required to fetch the possible triggering questions when
+        the question is being created. """
+        conditional_questions = self.filtered(lambda q: q.is_conditional)
+        non_conditional_questions = self - conditional_questions
+        non_conditional_questions.allowed_triggering_question_ids = False
+        non_conditional_questions.is_placed_before_trigger = False
+        if not conditional_questions:
+            return
+
+        possible_trigger_questions = self.search([
+            ('is_page', '=', False),
+            ('question_type', 'in', ['simple_choice', 'multiple_choice']),
+            ('suggested_answer_ids', '!=', False),
+            ('survey_id', 'in', self.survey_id.ids)
+        ])
+        # Using the sequence stored in db is necessary for existing questions that are passed as
+        # NewIds because the sequence provided by the JS client can be incorrect.
```

Comment:
As this creates some unusual code, I would be more confortable if it was covered by a tour (as it depends on both python computes and JS code).

Idea would be to create questions, move the around, make sure everything works properly.
You could even test deletion of questions triggering other questions etc.

(I think I mentioned that before, don't know if you have taken a look).